### DISCUSSION TITLE
GSOC 2024, Project 4

### DISCUSSION THREAD

**Discussion by chakrie1489** [MDAnalysis/mdanalysis#4497] [GSoC Discussions]
Hi team,

I'm Chakradhar Basani, currently a sophomore at IIT Gandhinagar, India, majoring in Material Science and Engineering and minoring in Computer Science and Engineering. I'm interested in the "2D visualization for small particles" project at MDAnalysis. 
Looking forward to working with you.

Thank you.

**Comment by RMeli** [comment]
Thank you for participating in GSOC 2024 with us. Whether you were selected as a GSoC contributor or not, you are a valued member of the MDAnalysis community and we are excited to work with you outside of the GSoC program as well!
